<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmViewLog</class>
 <widget class="QDialog" name="frmViewLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="txtLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
